frontend/src/hooks/review/useFormData.ts

```diff
@@ -0,0 +1,25 @@
+import type { ReviewRequest } from '@/types/review';
+
+interface useFormDataProps {
+  imageKey: string;
+  imageFile: File | null;
+  formContentKey: string;
+  formContent: ReviewRequest;
+}
+
+const useFormData = ({ imageKey, imageFile, formContentKey, formContent }: useFormDataProps) => {
+  const formData = new FormData();
+
+  if (imageFile) {
+    formData.append(imageKey, imageFile, imageFile.name);
+  }
+
+  const formContentString = JSON.stringify(formContent);
+  const formContentBlob = new Blob([formContentString], { type: 'application/json' });
+
+  formData.append(formContentKey, formContentBlob);
+
+  return { formData };
```

Comment:
```suggestion
  return formData;
```

formData만 반환한다면 하나만 반환하는건 어떤가요?